banzai_nres/frames.py

```diff
@@ -341,5 +402,14 @@ def open(self, path, runtime_context) -> Optional[ObservationFrame]:
         # products to distinguish.
         if image is None or 'nres' not in image.meta.get('TELESCOP', '').lower():
             return None
+
+        # Fix the RA and DEC keywords to be the requested values for all of our measurements
+        if image['TELESCOPE_1'].meta['OBJECT'].lower() in image.meta['OBJECTS'].lower():
+            telescope_num = 1
         else:
-            return image
+            telescope_num = 2
+        image.ra = image[f'TELESCOPE_{telescope_num}'].meta['CAT-RA']
+        image.dec = image[f'TELESCOPE_{telescope_num}'].meta['CAT-DEC']
```

Comment:
just to verify, image.ra now contains the correct (Requested) RA and DEC?

Reply:
Yes, that is indeed what this part of the code does.